exalearn-api/ExaLearn.WebApi/Migrations/20210720181133_db_filled_qa.cs

```diff
@@ -0,0 +1,520 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+namespace ExaLearn.WebApi.Migrations
+{
+    public partial class db_filled_qa : Migration
```

Comment:
why db_filled_qa create tables, migration in dev InitialDB already create them. Should be 2 migration InitialDB(dev has it), then FillDatabaseWithData

Reply:
> why db_filled_qa create tables, migration in dev InitialDB already create them. Should be 2 migration InitialDB(dev has it), then FillDatabaseWithData

Fixed